src/user/bans.js

```diff
@@ -43,8 +43,6 @@ module.exports = function (User) {
 		if (until > now) {
 			tasks.push(async.apply(db.sortedSetAdd, 'users:banned:expire', until, uid));
 			tasks.push(async.apply(User.setUserField, uid, 'banned:expire', until));
-		} else {
-			until = 0;
```

Comment:
this is never user after this point, so it's not needed

Reply:
It's not? I thought `until` is specified in the ban modal, and we wouldn't want to save something like `"NaN"` into the database... :confused:

Reply:
not in this code block, my IDE pointed it out for me, I didn't find it by just eye balling 😛